src/coffee/directives/api/models/parent/windows-parent-model.coffee

```diff
@@ -200,7 +201,17 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
                 @interpolateContent(@linked.element.html(), model)
             @DEFAULTS = @scopeOrModelVal(@optionsKey, @scope, model) or {}
             opts = @createWindowOptions gMarker, childScope, fakeElement.html(), @DEFAULTS
-            child = new WindowChildModel model, childScope, opts, @isIconVisibleOnClick, gMap, @markersScope?.plurals.get(model[@idKey])?.scope, fakeElement, false, true
+
+            child = new WindowChildModel {
```

Comment:
Explicit on options to create a WindowChildModel. This also allows you to not worry about argument order.